ui/src/app/assets/images/header-left-hexs.svg

```diff
@@ -0,0 +1,50 @@
+						c0.9-0.5,1.5-1.4,1.5-2.4v-8.4c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C96.6,25.9,96,26.8,96,27.8"/>
+					<path class="st1" d="M24,27.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C24.6,25.9,24,26.8,24,27.8"/>
+					<path class="st1" d="M0,27.8v8.4c0,1,0.6,1.9,1.5,2.4L9,42.8c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C0.6,25.9,0,26.8,0,27.8"/>
+					<path class="st1" d="M12,47.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C12.6,45.9,12,46.8,12,47.8"/>
+					<path class="st1" d="M36,47.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C36.6,45.9,36,46.8,36,47.8"/>
+					<path class="st1" d="M72,67.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C72.6,65.9,72,66.8,72,67.8"/>
+					<path class="st1" d="M96,67.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C96.6,65.9,96,66.8,96,67.8"/>
+					<path class="st1" d="M84,47.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C84.6,45.9,84,46.8,84,47.8"/>
+					<path class="st1" d="M48,67.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C48.6,65.9,48,66.8,48,67.8"/>
+					<path class="st1" d="M24,67.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C24.6,65.9,24,66.8,24,67.8"/>
+					<path class="st1" d="M0,67.8v8.4c0,1,0.6,1.9,1.5,2.4L9,82.8c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4v-8.4
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C0.6,65.9,0,66.8,0,67.8"/>
+					<path class="st1" d="M84,7.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4V7.8
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C84.6,5.9,84,6.8,84,7.8"/>
+					<path class="st1" d="M60,7.8v8.4c0,1,0.6,1.9,1.5,2.4l7.5,4.2c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4V7.8
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C60.6,5.9,60,6.8,60,7.8"/>
+					<path class="st1" d="M36,7v8.4c0,1,0.6,1.9,1.5,2.4L45,22c0.9,0.5,2,0.5,2.9,0l7.5-4.2c0.9-0.5,1.5-1.4,1.5-2.4V7
+						c0-1-0.6-1.9-1.5-2.4l-7.5-4.2c-0.9-0.5-2-0.5-2.9,0l-7.5,4.2C36.6,5.1,36,6,36,7"/>
```

Comment:
What does this do?

Reply:
This is an image that is laid on top of the "Job Manager" header; see it rendered [here](https://github.com/DataBiosphere/job-manager/blob/7f059301678fa7c34ebb2d1f99ffe28cd397ef6e/ui/src/app/assets/images/header-left-hexs.svg ).

Reply:
Wow, this is ..... awesome 😲